website/docs/Software/OpenBCISoftware/01-OpenBCI_GUI.md

```diff
@@ -319,8 +319,64 @@ If you are encountering issues launching the GUI at this point, please head to t
+| 5      | EXG Channel 3           | EEG/EMG/ECG channel connected to pins N4P                |
+| 6      | EXG Channel 4           | EEG/EMG/ECG channel connected to pins N5P                |
+| 7      | EXG Channel 5           | EEG/EMG/ECG channel connected to pins N6P                |
+| 8      | EXG Channel 6           | EEG/EMG/ECG channel connected to pins N7P                |
+| 9      | EXG Channel 7           | EEG/EMG/ECG channel connected to pins N8P                |
+| 10     | Accel Channel 0         | Accelerometer channel 0 (X)                              |
+| 11     | Accel Channel 1         | Accelerometer channel 1 (Y)                              |
+| 12     | Accel Channel 2         | Accelerometer channel 2 (Z)                              |
+| 13     | Other                   | See below                                                |
+| 14     | Other                   | See below                                                |
+| 15     | Other                   | See below                                                |
+| 16     | Other                   | See below                                                |
+| 17     | Other                   | See below                                                |
+| 18     | Other                   | See below                                                |
+| 19     | Other                   | See below                                                |
+| 20     | Analog Channel 0        | Analog channel 0                                         |
+| 21     | Analog Channel 1        | Analog channel 1                                         |
+| 22     | Analog Channel 2        | Analog channel 2                                         |
+| 23     | Timestamp               | Unix timestamp                                           |
+| 24     | Marker Channel          | Channel for adding manual markers to data                |
+| 25     | Timestamp (Formatted)   | Year-Month-Day Hour:Minute:Second (Not in Brainflow csv) |
+
+:::info Cyton Other Channels
 When recording using Cyton, the GUI will output a data file which contains 7 channels with the header "Other". These channels are user defined based on the [board mode](../../Cyton/04-OpenBCI_Cyton_SDK.md/#board-mode). Refer to the [aux data footer section](../../Cyton/03-Cyton_Data_Format.md/#firmware-version-200-fall-2016-to-now-1) of the Cyton data format documentation for further information.
 
-In digital read mode the 5th "Other" channel is connected to the D17 pin by default. This pin can be triggered by pressing the "PROG" button. This is useful for adding manual timestamps to your data.
\ No newline at end of file
+In digital read mode the 5th "Other" channel is connected to the D17 pin by default. This pin can be triggered by pressing the "PROG" button. This is useful for adding manual timestamps to your data.
+:::
+
+### Ganglion
```

Comment:
The Other channels still exist right now in the Ganglion recording and are unused. I think you removed them here, but we have to specify that they are labeled `other` and description == `not used`.